src/drivers/driver_zephyr.c

```diff
@@ -1356,7 +1356,8 @@ static int _wpa_drv_zep_set_key(void *priv,
 				const u8 *seq,
```

Comment:
Also in commit `Add key_flag parameter for set_key`, it is missing commit body that should describe why this change is done/needed. As a general note, the commit body should not be left empty.

Reply:
updated, pls review again